resources/views/navigation-menu.blade.php

```diff
@@ -1,4 +1,10 @@
 @php
+
+// Decide at what point the mobile menu should be displayed
+// Both variables are necessary for the Tailwind compilation to work
```

Comment:
Are you sure about that? Why would Tailwind care at all about PHP code?

Reply:
Because Tailwind needs to have the full class names in order to generate them properly. Constructing class names dynamically will prevent them from being generated.
See https://tailwindcss.com/docs/content-configuration#dynamic-class-names